kbmetamodel/src/main/java/de/kernbeisser/MetaModelProcessor.java

```diff
@@ -0,0 +1,121 @@
+package de.kernbeisser;
+
+import com.google.auto.service.AutoService;
+import jakarta.persistence.Entity;
```

Comment:
jakarta und javax gemischt zu sehen ist recht ungewöhnlich...
Kannst du mir das vielleicht in zwei oder deri Sätzen erklären?

Reply:
Oh eigentlich sollte kein Javax mehr da sein :/ Hibernate 6 ist ja auf Jakarta umgestiegen, deshalb sollte eigentlich nur noch Jakarta genutzt werden.